nautobot/dcim/api/serializers.py

```diff
@@ -199,6 +199,18 @@ class Meta:
         list_display_fields = ["name", "status", "parent", "tenant", "description", "tags"]
         # https://www.django-rest-framework.org/api-guide/validators/#optional-fields
         validators = []
+        detail_view_config = [
+            {
+                "Location": {"fields": ["name", "display", "rack_count", "parent"]},
+                "Management": {"fields": ["status", "prefix_count", "device_count"]},
+                "Others": {"fields": ["tenant", "natural_key_slug", "url"]},
+            },
+            {
+                "Tags": {"fields": ["tags"]},
+                "Comments": {"fields": ["comments"]},
+                "Contact": {"fields": ["contact_name", "contact_phone", "contact_email"]},
+            },
```

Comment:
Shouldn't at least some of these have `template: list`?

Reply:
As of now, just the list template can be defined; I don't see the need in explicitly declaring it again.

Reply:
OK, just seems asymmetrical to me that `view_config()` returns a list of dicts with `fields` and `template` as keys, while a statically declared `detail_view_config` only contains `fields`?